src/test/java/com/amazon/opendistroforelasticsearch/security/SlowIntegrationTests.java

```diff
@@ -159,4 +165,24 @@ public void testNodeClientDisallowedWithNonServerCertificate2() throws Exception
         }
     }
 
+    @Test
+    public void testDelayInSecurityIndexInitialization() throws Exception {
+        final Settings settings = Settings.builder()
+                .put(ConfigConstants.OPENDISTRO_SECURITY_ALLOW_DEFAULT_INIT_SECURITYINDEX, true)
+                .put("cluster.routing.allocation.exclude._ip", "127.0.0.1")
+                .build();
+        try {
+            setup(Settings.EMPTY, null, settings, false);
+        } catch (IOException e) {
```

Comment:
If this exception is expected to be thrown, add `fail()` after `setup()`.

Reply:
Done